packages/validator/lib/validator.js

```diff
@@ -45,4 +47,14 @@ validators.isSlug = function isSlug(str) {
     return validators.matches(str, /^[a-z0-9\-_]+$/);
 };
 
+validators.isEmail = function isEmail(str, options = {}) {
+    assertString(str);
+    // Use the latest email validator if the option is set
+    if (options?.latest) {
```

Comment:
@ErisDS The only thing i'd like to get a sense check on this PR is naming for the option that determines if the new email validator should be used instead of default.

Reply:
I'd like to leave an issue open to clean this up, so we go through and change all the existing calls to explicitly declare they use the old version. 

So I'd probably reverse this concept and have the option be called `legacy` and have it default to true? 

So your call will do `isEmail('xxx', {legacy: false})`.

We can then later change the default to be `legacy: false` which makes more sense long term?

Reply:
Yup that's way better, I wasn't warming up to `latest`! 🙏  I'll clean this up now and add an issue for us to clean up `legacy` usage across codebase in future.

Reply:
Updated